cli.js

```diff
@@ -693,8 +695,8 @@ let handled = false;
 yargs
   .scriptName('xrpk')
   .command('whoami', 'print logged in address', yargs => {
-    yargs
-      /* .positional('input', {
+    yargs;
```

Comment:
This was clearly meant to subsume the below commented code. Adding this semicolon doesn't help anything and we shouldn't be adding weird formatting just to please a linter.

Reply:
If a linter rule is creating worse code then we shouldn't have the linter rule.

Or, we should change the code to make the linting make more sense.

Reply:
I've commented out the entire block and reverted the semicolon change